base/monitoring/kfdef.yaml

```diff
@@ -18,6 +18,11 @@ spec:
           name: manifests
           path: prometheus/cluster
       name: prometheus-cluster
+    - kustomizeConfig:
```

Comment:
What OCP version is the cluster on?

Reply:
Uf.. I don't understand how that's related to this line. I think base should target any ODH supported OCP version, why?

Reply:
If it's on a new enough version of OpenShift the ODH operator prometheus CRDs will conflict with the OpenShift cluster monitoring CRDs. So subscribing to the operator manually _could_ cause issues.

Reply:
I'm not adding anything new here, it's moved from the jupyterhub spec, since we should keep it rather here, WDYT?

https://github.com/operate-first/odh/pull/46/files#diff-f189210e84717543c6430664a9d2c1fa5eee8553dc0450d15b29a4ea5651bcceL18-L26

Reply:
yeah leave this for now, this is a topic for another pr